wrappers/samtools/fastq/fastq_prep.py

```diff
@@ -80,6 +80,38 @@ def concatenate_fastq(r1, r2, family, sample):
         log_message(f"Input {r1}, {r2} given for {sample} is not handled. Exiting!")
         exit()
 
+def decompress_ora(r1,r2):
+    decompressed_r1_list=[]
+    decompressed_r2_list=[]
+    for read in r1:
+        log_message(f"{read}")
+        command = f"/hpf/largeprojects/ccmbio/ajain/tools/orad_2_6_1/orad --ora-reference /hpf/largeprojects/ccmbio/ajain/tools/orad_2_6_1/oradata {read}"
```

Comment:
Instead hard-coding the ora variables, can you include the path to the orad executable and ora-reference as a snakemake parameters and retrieve those values from the config?

Reply:
Sure, that's a good point! I have a few jobs running on my crg2 branch right now so I'll make the requested edits and push the changes early next week.